crates/solidity/outputs/cargo/tests/Cargo.toml

```diff
@@ -22,6 +22,7 @@ slang_solidity = { workspace = true, features = [
   "__private_ariadne_errors",
   "__private_testing_utils",
   "__private_backend_api",
+  "__private_wasm_apis",
```

Comment:
Check: this shouldn't be necessary, probably a merge that went wrong.

Reply:
Ah, this is OK: when run `infra test cargo` it uses all features, but when running the specific package with `cargo test` I see the error from not having this feature enabled. It's better to be explicit of the features needed by the test.